lib/iris/_concatenate.py

```diff
@@ -235,7 +235,7 @@ class _CoordExtent(namedtuple('CoordExtent',
     __slots__ = ()
 
 
-def concatenate(cubes, error_on_mismatch=False):
+def concatenate(cubes, check_aux_coords=True, error_on_mismatch=False):
```

Comment:
please may you reverse the order of these arguments, in case someone has used error_on_mismatch positionally